custom_components/econnect_metronet/devices.py

```diff
@@ -72,7 +63,7 @@ def __init__(self, connection, config=None):
         self._sectors_vacation = config.get(CONF_AREAS_ARM_VACATION) or []
 
         # Alarm state
-        self.state = STATE_UNAVAILABLE
+        self.state = None
```

Comment:
I took time to deep dive in Home Assistant source code, and it looks like there are no ways to use `STATE_UNAVAILABLE` without triggering unintentional future problems. Given that, I would say that using `None` is the right thing to do. I may want to double check on production if it has any side effect.

Did you notice something after using in Home Assistant your changes?

Reply:
No, everything was looking fine (I assume that since the device is `update`ing at startup, chances that state is `STATE_UNAVAILABLE` are very little to begin with.
But to be honest, in my setup I am running your latest release, so I didn't do too much testing.
I will play with it a little bit more and get back more feedback.
Is there any particular case you want me to stress test?

Reply:
I think we can proceed and merge your PR. Worst case we'll just make another change to fix any unexpected behavior. The only thing I would like to test is what happens when the device goes unavailable for real (e.g. the alarm device is disconnected from the Internet).

But I guess we're just testing a new behavior due to HA changes rather than your changes. Worth the check though! In the meantime I would resolve this conversation and proceed with the merge!